test/cli-test.js

```diff
@@ -83,14 +101,17 @@ describe('Mustache CLI', function () {
 
     it('writes it couldnt find template into stderr when second argument doesnt resolve to a file', function(done) {
       exec('bin/mustache test/_files/cli.json test/_files/non-existing-template.mustache', function(err, stdout, stderr) {
-        assert.notEqual(stderr.indexOf('Could not find file: test/_files/non-existing-template.mustache'), -1);
+        console.log(stderr);
+        assert.notEqual(stderr.indexOf('Could not find file:'), -1);
+        assert.notEqual(stderr.indexOf(changeForOS('test/_files/non-existing-template.mustache')), -1);
```

Comment:
I see. That's not very obvious, just by reading the code..

I was going to suggest using a regex, eg `/Could not find file.*non-existing-template\.mustache/`., but don't worry about it, I'm going to refactor these assertions after we merge this PR (using `unexpected` or some nice assertion library).